### PR TITLE
fix(ios-angular): Topmost deprecation and not rendering on top of modal

### LINKED ISSUE #60: Datepicker not showing inside a secondary modal
I'm using nativescript-angular. When I call the datepicker inside a modal on top of a modal it wont show.

### PR DESCRIPTION
**Pull request by bbroereES**
## PR Checklist

- [x] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [x] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [x] All existing tests are passing
- [ ] Tests for the changes are included

## What is the current behavior?
iOS gives a warning about deprecation of topmost() and does not render the datetimepicker on top of a modal in Angular.

## What is the new behavior?
iOS gives no warning about deprecation of topmost() and does render the datetimepicker on top of a modal in Angular.

Fixes/Implements/Closes #60.

### CODE REVIEW

**Comment by bbroereES** [comment]
@cla-bot check

**Comment by NathanWalker** [comment]
Thanks @bbroereES !

**Review by NathanWalker — APPROVED** [review]

**Comment by jscti** [comment]
@NathanWalker Could this be merged ?

**Comment by lochstar** [comment]
I also need this.

**Comment by NathanWalker** [comment]
@lochstar and @jscti forgive us for the delay. I went ahead and integrated your fix on this PR just to avoid further delay:
https://github.com/NativeScript/plugins/pull/10/files#diff-aad3f150e3a162d27727560b5fa63351R146

That included some formatting to match the style in the workspace but highlighted the diff which contains your change there.
Frame.topmost() adjustment was already in there with the {N} 7 updates.

Published in `2.0.2` now.

In future you can PR any changes to the plugin workspace where we manage all @nativescript scope packages:
https://github.com/NativeScript/plugins

**Comment by jscti** [comment]
@NathanWalker 
Thanks ;)

But I don't get quite how to use it ?
Since then I used "tns plugin add nativescript-datetimepicker" (last version : 1.2.2)
Now should I use "tns plugin add @nativescript/datetimepicker" (last version today 2.0.2) ? But there is no "NativeScriptDateTimePickerModule" to import

tldr: how to use your fix by "tns plugin add" it ?

edit: Well, I found the proper import :
`import { NativeScriptDateTimePickerModule } from '@nativescript/datetimepicker/angular';`
But it's leading to a "Error: No NgModule metadata found for 'undefined'." ...
I'm still on NS6.X
edit2: well new repo is only for NS7 ...

**Comment by jscti** [comment]
@NathanWalker Isn't it possible to merge this PR in this repo as well for NS6 ?

I can't upgrade to NS7 (too much plugins are dead ... ) and I'm struggling forking/building/tns adding this one PR to make it work ... It's a  real pain
